### ISSUE TITLE
[FEAT]Class Imbalance check

### ISSUE THREAD

**Issue by TheSolY** [deepchecks/deepchecks#1607]
Add a data integrity (single dataset) check for class imbalance, relevant for both tabular and vision
The check calculate the ratio between the class frequencies, for a multiclass case, the ratio between the most prevalent and the least prevalent.
This check should have a condition to alert if the ratio is above a certain threshold.

**Comment by shrey231** [comment]
Hi! I was wondering if I could be assigned to this issue?

**Comment by TheSolY** [comment]
@shrey231 Hi, we appreciate your willingness to contribute! I edited the issue a bit to make it clearer and assigned you :)

**Comment by shrey231** [comment]
Sounds good thank you so much! I was wondering if there is a specific library thats being used for data vis (for displaying counts per class).

**Comment by matanper** [comment]
Yes, we use plotly, you can take examples from the other checks' source code

**Comment by shrey231** [comment]
Sounds good thank you!

**Comment by DanBasson** [comment]
hi @shrey231 
can i take this issue?

**Comment by shrey231** [comment]
Yes, unfortunately I do not have enough time to complete this issue. Thank you!

**Comment by DanBasson** [comment]
thanks @shrey231 .

@TheSolY can you please assign this issue to me?
